src/domain/event/ExternalTicketSystemEventIsEnrolled.tsx

```diff
@@ -2,28 +2,28 @@ import { useTranslation } from 'react-i18next';
 import { useParams } from 'react-router-dom';
 import { useQuery } from '@apollo/client';
 
-import styles from './ticketmasterEventIsEnrolled.module.scss';
-import { ticketmasterEventQuery } from './queries/ticketmasterEventQuery';
+import styles from './externalTicketSystemEventIsEnrolled.module.scss';
+import { externalTicketSystemEventQuery } from './queries/externalTicketSystemEventQuery';
 import {
-  ticketmasterEventQuery as TicketmasterEventQueryType,
+  externalTicketSystemEventQuery as ExternalTicketSystemEventQueryType,
   // eslint-disable-next-line max-len
-  ticketmasterEventQuery_event_ticketSystem_TicketmasterEventTicketSystem as EventTicketSystem,
-} from '../api/generatedTypes/ticketmasterEventQuery';
+  externalTicketSystemEventQuery_event_ticketSystem_TicketmasterEventTicketSystem as EventTicketSystem,
```

Comment:
Doesn't this assume that `externalTicketSystemEventQuery_event_ticketSystem_TicketmasterEventTicketSystem` is the same interface as `externalTicketSystemEventQuery_event_ticketSystem_LippupisteEventTicketSystem` for this file's code to work correctly? If so, then should this dependency perhaps be made explicit by combining `externalTicketSystemEventQuery_event_ticketSystem_TicketmasterEventTicketSystem` and `externalTicketSystemEventQuery_event_ticketSystem_LippupisteEventTicketSystem` into one or some other way?

Reply:
well, they are the same, but it definitely needs to point some where else, in an union type like you said. I'll fix this.